orc8r/cloud/go/swagger/combine.go

```diff
@@ -0,0 +1,197 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger
+
+import (
+	"regexp"
+	"sort"
+
+	"magma/orc8r/cloud/go/tools/swaggergen/generate"
```

Comment:
See above comment -- this PR should include the changes required such that application code is not importing from tool code. While this might not immediately break things, we should treat this lib vs. tool invariant as a MUST NOT rather than a SHOULD NOT

Ref for the difference between these two: https://tools.ietf.org/html/rfc2119